### PR TITLE
Draft agri services

### PR DESCRIPTION
**Pull request by AbhijeetONDC**
Fixed according to previous MR comments

### CODE REVIEW

File: api/components/Examples/Agriculture_services/search/search_by_item.json
Comment (tanyamadaan) [inline api/components/Examples/Agriculture_services/search/search_by_item.json:1]: only live search is required in agri services?

File: api/components/Examples/Agriculture_services/search/search_by_item.json [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+        ]
+      },
+      {
+        "descriptor": {
+          "code": "BAP_DETAILS"
+        },
+        "list": [
+          {
+            "descriptor": {
+              "code": "BAP_TYPE"
```
Comment (tanyamadaan) [inline api/components/Examples/Agriculture_services/search/search_by_item.json:78]: Farmer is a BAP?

File: api/components/Examples/Agriculture_services/search/search_by_item.json [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+        }
+      },
+      "fulfillment": {
+        "type": "Seller-Pickup",
+        "stops": [
+          {
+            "type": "end",
+            "time": {
+              "range": {
+                "start": "2024-04-04T22:00:00.000Z"
```
Comment (tanyamadaan) [inline api/components/Examples/Agriculture_services/search/search_by_item.json:35]: what does this range.start denotes?

File: api/components/Examples/Agriculture_services/search/search_by_item_assaying.json [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+  "message": {
+    "intent": {
+      "item": {
+        "descriptor": {
+          "name": "Arahar Whole Assaying Services"
+        }
+      },
+      "fulfillment": {
+        "type": "Seller-Pickup",
+        "customer": {
```
Comment (tanyamadaan) [inline api/components/Examples/Agriculture_services/search/search_by_item_assaying.json:30]: why is customer number required in search?

File: api/components/Examples/Agriculture_services/search/search_by_item_assaying.json [hunk 4]
```diff
@@ -0,0 +1,212 @@
...
+              "descriptor": {
+                "code": "BUYER_FINDER_FEE_AMOUNT"
+              },
+              "value": "0"
+            }
+          ]
+        },
+        {
+          "descriptor": {
+            "code": "COMMODITY_DETAILS"
```
Comment (tanyamadaan) [inline api/components/Examples/Agriculture_services/search/search_by_item_assaying.json:80]: what is the need of this? when the display is also false.

**Review by tanyamadaan — CHANGES_REQUESTED** [review]

**Comment by tanya-ondc** [comment]
Merging it as requested by Sumit.

**Pull request merged**